### PR TITLE
Changelog for 0.18.0

### PR DESCRIPTION
**Pull request by angelozerr**
Changelog for 0.18.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+- Support customized template path for @CheckedTemplate. See [quarkus-ls#893](https://github.com/redhat-developer/quarkus-ls/issues/893).
+- Resolve system properties/environment variables while browsing the application.properties values. See [lsp4mp#448](https://github.com/eclipse/lsp4mp/issues/448).
+- Add support for @Startup healthcheck diagnostic. See [lsp4mp#443](https://github.com/eclipse/lsp4mp/issues/443).
+
+### Bug Fixes
+
+- qute(NoMatchingTemplate) if using @CheckedTemplate with basePath. See [#787](https://github.com/redhat-developer/vscode-quarkus/issues/787).
+- Qute: Cannot locate hyphenated template name. See [quarkus-ls#975](https://github.com/redhat-developer/quarkus-ls/issues/975).
+- Check if record implement TemplateInstance to provide the support. See [quarkus-ls#974](https://github.com/redhat-developer/quarkus-ls/pull/974).
+- fix: qute(NoMatchingTemplate) if using @CheckedTemplate with basePath. See [quarkus-ls#972](https://github.com/redhat-developer/quarkus-ls/pull/972).
```
Comment (fbricon) [inline CHANGELOG.md]: duplicate of L17

Comment (angelozerr) [reply CHANGELOG.md]: removed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+- fix: synchronized projectLabels + collect codeLens when project is loaded. See [lsp4mp#444](https://github.com/eclipse/lsp4mp/pull/444).
+
+### Build
+
+- Migrate Jenkins release job to GitHub Actions. See [#723](https://github.com/redhat-developer/vscode-quarkus/pull/723).
+
+### Other
+
+- Hide `Debug Quarkus` command from palette. See [#743](https://github.com/redhat-developer/vscode-quarkus/issues/743).
+- Use category `Quarkus` for all commands. See [#741](https://github.com/redhat-developer/vscode-quarkus/issues/741).
```
Comment (datho7561) [inline CHANGELOG.md]: These can go under bug fixes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,50 @@
...
+- Bump gulp from 4.0.2 to 5.0.0. See [#713](https://github.com/redhat-developer/vscode-quarkus/pull/713).
+- Replace vulnerable pom-parser with fast-xml-parser in tests, and other updates. See [#657](https://github.com/redhat-developer/vscode-quarkus/pull/657).
+- Replace axios library with built-in fetch. See [#711](https://github.com/redhat-developer/vscode-quarkus/pull/711).
+- Fix tests for release workflow. See [vscode-microprofile#185](https://github.com/redhat-developer/vscode-microprofile/pull/185).
```
Comment (datho7561) [inline CHANGELOG.md]: I think we should remove these entries, since they aren't interesting to the end user.

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,49 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## [0.18.0](https://github.com/redhat-developer/vscode-quarkus/milestone/20?closed=1) (August 26, 2024)
```
Comment (rgrunber) [inline CHANGELOG.md:3]: @angelozerr (first) 0 -> 1 .

Comment (datho7561) [reply CHANGELOG.md:3]: aaa good catch

Comment (angelozerr) [reply CHANGELOG.md:3]: It is too bad -( I cannoyt fix that since I created the release -( 

Good catch @rgrunber !

**Pull request merged**